phy/utils/dock.py

```diff
@@ -266,12 +273,27 @@ def _try_enable_ipython_qt():
     if not _is_interactive():
         return False
     if ip:
-        qt_app = ip.enable_gui('qt')
-        if qt_app:
-            return qt_app
+        ip.enable_gui('qt')
+        global _APP_RUNNING
+        _APP_RUNNING = True
+        return True
     return False
 
 
+def enable_qt():
+    if not _check_qt():
+        return
+    try:
+        from IPython import get_ipython
+        ip = get_ipython()
+        ip.enable_gui('qt')
+        global _APP_RUNNING
+        _APP_RUNNING = True
+        info("Qt event loop activated.")
+    except:
+        warn("Qt event loop not activated.")
```

Comment:
not necessarily. qt event loop integration is horribly hard to get right: it needs to work in 875877 cases...